starboard/extension/setstring_test.cc

```diff
@@ -0,0 +1,69 @@
+// Copyright 2024 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This is a kind of death test, as described in
+// https://github.com/google/googletest/blob/main/docs/advanced.md#death-tests,
+// except, it doesn't spawn another process, but terminates itself.
+// It is intended to be started from a host machine on a target device
+// to generate a coredump, which is verified on the host.
```

Comment:
This test doesn't actually do any verifications about the expected annotations in the core dump. Will those be added later? Or is the expectation that the core dump, and its annotations, will be verified manually after this test is run?

Reply:
Such happened, target part of the test is cross-platform. But verification code, apparently, can be platform-specific only. I've implemented it for PlayStation, but I see no way to implement similar for UWP/Windows. This is current status.

Reply:
I support getting the code in, if it helps bringing up integration tests.